### PR TITLE
BXMSDOC-2608-master: Added the asynchronous signal events section in the document.

### PR DESCRIPTION
**Pull request by kaldesai**
- [Dedicated JIRA](https://issues.redhat.com/browse/BXMSDOC-2608)
- [Managing and monitoring business processes in Business Central](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-2608-RHPAM-Async-Sig/#asynchronous-signal-events-con)

Can you please review and verify the sections in the document?

- **3.8. Asynchronous signal events** 
- **3.8.1. Configuring asynchronous signals for intermediate events**
- **3.8.2. Configuring asynchronous signals for end events**

This information was present in the 6.4 documents. I have tweaked and adjusted in this document.
Please provide feedback on the content.

### CODE REVIEW

File: doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[id='asynchronous-signal-events-con']
+
+= Asynchronous signal events
+
+In some cases where several process instances from different process definitions are waiting for the same signal. They are generally executed sequentially in the same single thread. But, if one of those process instances throws a runtime exception, all the other process instances are affecte and usually result in a rolled back transaction. To avoid this situation, {PRODUCT} supports using asynchronous signals events for:
+
+* Throwing intermediate signal events
+* End events
+
+In {CENTRAL}, set the *Data Input* value of the throw event to *async* to automatically set the executor service on each session. This ensures that each process instance is signaled in a different transaction.
```
Comment (jstastny-cz) [inline doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc:10]: Please provide more info here:
* changes are supposed to be made in Process Designer
* the procedure is done in the "Data assignments" section of Properties panel
* Add a data input property with name 'async', other values leave blank.

Comment (kaldesai) [reply doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc:10]: Done

File: doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[id='asynchronous-signal-events-con']
+
+= Asynchronous signal events
+
+In some cases where several process instances from different process definitions are waiting for the same signal. They are generally executed sequentially in the same single thread. But, if one of those process instances throws a runtime exception, all the other process instances are affecte and usually result in a rolled back transaction. To avoid this situation, {PRODUCT} supports using asynchronous signals events for:
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc:5]: Done

File: doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[id='asynchronous-signal-events-con']
+
+= Asynchronous signal events
+
+In some cases where several process instances from different process definitions are waiting for the same signal. They are generally executed sequentially in the same single thread. But, if one of those process instances throws a runtime exception, all the other process instances are affected and usually result in a rolled back transaction. To avoid this situation, {PRODUCT} supports using asynchronous signals events for:
+
+* Throwing intermediate signal events
+* End events
+
+In the process designer of {CENTRAL}, set the data input value of the throw event to *async* in the *Data assignments* section of *Properties* panel leaving other values blank. It will automatically set the executor service on each session. This ensures that each process instance is signaled in a different transaction.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc:10]: Suggested rewrite:
To ensure that each process instance is signaled to a different transaction, set the data input value of the throw event as async in the Data assignments of the Properties panel. This sets the executor service in each session.

File: doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[id='asynchronous-signal-events-con']
+
+= Asynchronous signal events
+
+In some cases where several process instances from different process definitions are waiting for the same signal. They are generally executed sequentially in the same single thread. But, if one of those process instances throws a runtime exception, all the other process instances are affected and usually result in a rolled back transaction. To avoid this situation, {PRODUCT} supports using asynchronous signals events for:
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc:5]: Suggested rewrite:
When several process instances from different process definitions are waiting for the same signal, they are executed sequentially in the same thread.

File: doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc (hunk 3 above)
Comment (kaldesai) [reply doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc:10]: Done

File: doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc (hunk 4 above)
Comment (kaldesai) [reply doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc:5]: Done and thank you.

**Pull request merged**